Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -371,17 +367,17 @@ function NotNullAssert(v: unknown): asserts v is NotNull {}
         const informationPopUp = new BocAutoCompleteReferenceValue_InformationPopUp(options, input);
         let blockSubmit: Optional<boolean>;
 
-        $input.bind("keydown.autocomplete", function(event) {
+        $(input).bind("keydown.autocomplete", function(event) {
```

Comment:
Replacing the event wiring will be done together with replacing the jquery-hookup?

Reply:
Yes this will be done in the follow up task